### ISSUE TITLE
A11Y: Icons in your table should be using proper alt text

### ISSUE THREAD

**Issue by shawnthompson** [esdc-devcop/guides#9]
On the page [Working With Git - Best Practices](https://esdc-devcop.github.io/guides/source-control/working-with-git.html)

You should be using descriptive alternative text for your icons in the table.

```
<img class="emoji" title=":heavy_check_mark:" alt=":heavy_check_mark:" src="https://github.githubassets.com/images/icons/emoji/unicode/2714.png" width="20" height="20">
```

**Comment by s-laugh** [comment]
Transferring as doc has moved to [Git Branching, Picking a Git Flow](https://esdc-devcop.github.io/recommendations/source_management/git-branching.html)
